src/main/java/org/fogbowcloud/arrebol/core/models/task/Task.java

```diff
@@ -78,19 +54,20 @@ public void setState(TaskState newState) {
         this.state = newState;
     }
 
-    public List<Command> getAllCommands() {
+    public List<Command> getCommands() {
         return commands;
     }
 
-    public String getId() {
-        return this.id;
+    public void putMetadata(String attributeName, String value) {
+        this.metadata.put(attributeName, value);
     }
 
-    public int getRetries() {
-        return this.retries;
+    public String getMetadata(String attributeName) {
```

Comment:
I prefer to remove this method. The client should be responsible to get the value based on a key

Reply:
Done.